app/helpers/users_helper.rb

```diff
@@ -17,8 +17,8 @@ def can_view_funding?
     FeatureService.enabled?("funding") && defined?(current_user) && !current_user&.system_admin
   end
 
-  def can_view_upload_funding?
-    FeatureService.enabled?("upload_funding") && defined?(current_user) && current_user&.system_admin
+  def can_view_funding_uploads?
```

Comment:
is this still used? if this is within the system admin area, the authorisation will be covered by the route constraint.

Reply:
Ah wait, this is to feature flag the funding upload stuff specifically from support users as it's incomplete

Reply:
ah ok. maybe we can just inline the `FeatureService.enabled?("upload_funding")` then?

Reply:
Wait sorry it's definitely redundant. We do the same thing but for any feature flag with an existing method https://github.com/DFE-Digital/register-trainee-teachers/blob/99693c05125ff0d20561c047a45709e7e4386b2e/app/controllers/system_admin/funding_uploads_controller.rb#L5 which calls https://github.com/DFE-Digital/register-trainee-teachers/blob/3424e4731bb61e5f89de82f1c5de698b30fcc3d7/app/controllers/application_controller.rb#L126. Not sure why it was added, have removed it now.